test/db_root_command_shell_test.go

```diff
@@ -232,6 +232,21 @@ func (s *DBRootCommandShellSuite) Test_GivenATableWithRecords_WhenCreateIndexAnd
 	s.tc.AssertSqlEquals(outS, expected)
 }
 
+func (s *DBRootCommandShellSuite) Test_GivenATableWithRecords_WhenInsertValueWithSingleQuoteAndCallDotDumpCommand_ExpectNoErrors() {
```

Comment:
```suggestion
func (s *DBRootCommandShellSuite) Test_GivenATableWithRecordsWithSingleQuote_WhenCalllDotDumpCommand_ExpectSingleQuoteScape() {
```
Explanation: I see the INSERT part as the preparation for the test, iow, the state that we're testing. This idea reflects into your test when `CreateEmptySimpleTable` and `INSERT` command is on the same block. Also, the result of the test is more than just "no errors", but the single quote being escaped

Reply:
Makes sense!